src/pages/characters.tsx

```diff
@@ -1,79 +1,114 @@
+            </div>
           </div>
-        ))}
+        )}
       </div>
-      {/* <div className="grid grid-cols-4">
-        {typedCostumes.map(costume => (
-          <div key={costume.id} className="flex items-center" onClick={()=>setCostume(costume)}>
-            <img src={costumeIDToIconURL(costume.id)} alt=""/>
-            {costume.character} ({costume.name.en})
-          </div>
-        ))}
-      </div> */}
-
-      <CharacterInfo character={currentCostume}></CharacterInfo>
-      {/* <div className="relative self-center h-32">
-        <div className="flex">
-          {firstRow.map((char) =>
-            <div className="iso-bg" >
-              <img style={{
-              minWidth: '74px',
-              maxWidth: '74px',
-              minHeight: '74px',
-              maxHeight: '74px',
-            }} title={char} src={characterToIconURL(char)} key={char}/>
+      <div className="absolute flex" style={{
+        left: '-40px',
+        top: '42px',
+      }}>
+        {secondRow.map((costume) =>
+          <div className="relative" onClick={() => {
+            setCostume(costume)
+          }}>
+            <div className={`absolute text-sm top-20 right-0 flex justify-center text-white ${costume.character == currentCostume.character ? '' : 'text-opacity-60'}`} style={{ width: '56px' }}>
+              {costume.character}
             </div>
-          )}
-        </div>
-        <div className="absolute flex" style={{
-          left: '-40px',
-          top: '40px',
-        }}>
-          {secondRow.map((char) =>
-            <div className="iso-bg" >
+            <div className={`iso-bg ${costume.character == currentCostume.character ? 'active' : ''}`} >
               <img style={{
-              minWidth: '74px',
-              maxWidth: '74px',
-              minHeight: '74px',
-              maxHeight: '74px',
-            }} title={char} src={characterToIconURL(char)} key={char}/>
+                minWidth: '74px',
+                maxWidth: '74px',
+                minHeight: '74px',
+                maxHeight: '74px',
+              }} title={costume.character} src={costume.iconURL} key={costume.character} />
             </div>
-          )}
+          </div>
+        )}
+      </div>
+    </div>
+  )
+}
+
+function CharacterCostumes({
+  currentCostume,
+  setCostume,
+}: {
+  currentCostume: CostumeInfo,
+  setCostume: Dispatch<SetStateAction<CostumeInfo>>,
+}) {
+  return (
+    <div className="flex flex-wrap justify-center gap-2 my-2">
+      {typedCharacters.get(currentCostume.character).map(costume => (
+        <div key={costume.id}
+          className={`flex items-center border-beige border-2 p-2 ${currentCostume.id == costume.id ? '' : 'border-opacity-30'}`}
+          onClick={() => setCostume(costume)}>
+          {costume.name.en}
         </div>
-      </div> */}
+      ))}
+    </div>
+  )
+}
+
+export default function Home() {
```

Comment:
I forgot to rename this component, we can probably rename it to CharactersPage or something along those lines?

Reply:
I'll do that, but it's not important because it's the default export

Reply:
It will show "Home" as the component name within the React Devtools so still a nice thing to fix imo